src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -248,10 +248,12 @@
         {%- endif -%}
 
         {{ widget|raw }}
-        <label{% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}>
-            {{- label is not same as(false) ? (translation_domain is same as(false) ? label : label|trans({}, translation_domain)) -}}
-            {{- form_errors(form) -}}
```

Comment:
i think it's intended to still render errors here, which is up for debate in general i believe :) 

for now this reverts #26591 which im not sure is expected.

Reply:
To show the error inside the label as well?

Could you suggest another way to not show the `<label>` when it was set to false?

Reply:
> To show the error inside the label as well?

Yep :) see #24435, #26167


cc @Nyholm

Reply:
> Make sure errors are a part of the label on bootstrap 4 - this is a requirement for WCAG2

Ok clear 👍 Let's make sure that rule gets preserved. Then the question would be whether a label still should get rendered even when `false` gets passed.

The use-case here is that we want to render a custom label, while we want to keep using the Bootstrap theme for rendering the widget.

The problem is that `{{ form_widget(} }}` does render both the widget and the label. By passing `false` we basically say that userland is responsible for complying to the WCAG2 that the error should be rendered inside the label.

Reply:
i once suggested to make the split explicit, i.e. `form_label` + `form_label_with_errors`

> {{ form_widget(} }} does render both the widget and the label

this sounds like a violation of concerns already :)

im not sure what path to take. I agree a user could be responsible, and IMHO SF should provide low-level building blocks for it.

Reply:
Can't we fix the initial point for now? Display the label if there are errors?

Reply:
Both when a value or `false` is passed for `label` the errors do get rendered. Rendering the errors in the label according to WCAG2 should get preserved.

The problem for userland `ChoiceType` implementations; when `label => false` is passed, the label value does not get rendered while the `<label>` tag does get rendered.

So the change I suggest via this PR is to also not render the `<label>` tag when `false` is passed. The error message will then bubble up to the parent.

Reply:
Are you sure that errors bubble up?

Reply:
friendly ping @genexp

Reply:
friendly ping @rvanlaak

Reply:
Have been working on a unit test to verify this, but did not manage to complete that yet.